steps/testlio-upload/0.1.0/step.yml

```diff
@@ -0,0 +1,63 @@
+title: |-
+  Testlio Upload
+summary: |
+  Uploads IPA to Testlio
+description: |
+  Uploads IPA to Testlio to perform test case validation
+website: https://github.com/dmiluski/bitrise-step-testlio-upload
+source_code_url: https://github.com/dmiluski/bitrise-step-testlio-upload
+support_url: https://github.com/dmiluski/bitrise-step-testlio-upload/issues
+published_at: 2021-09-03T20:348:19-07:00
+source:
+  git: https://github.com/dmiluski/bitrise-step-testlio-upload.git
+  commit: 428ca8b0f6f0001181110fbcbd1b4e9f142ebd71
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+project_type_tags:
+  - ios
+  - macos
+type_tags:
+  - utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: true
+run_if: ""
+inputs:
+  - opts:
+      description: |
+        This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+      is_required: true
+      summary:
+        The app file you want to upload to BrowserStack, usually $BITRISE\_IPA\_PATH
+      title: A Bitrise generated IPA path
+    upload_path: $BITRISE_IPA_PATH
+  - testlio_api_token: $TESTLIO_API_TOKEN
+    opts:
+      description: |
+        Authentication to the API is performed via ​JWT​.
+      is_required: true
+      is_sensitive: true
+      summary: Your Testlio API token
+      title: Testlio API Token
+
+  - testlio_collection_id: $TESTLIO_COLLECTION_ID
```

Comment:
Default values should not use such env. variables.
They are only permitted if normally available in environment (eg. exported by anther step, bitrise CLI or bitrise.io).

Reply:
Hi @koral-- ,

I was referencing: [AppCenter IPA upload](https://github.com/kanari3/bitrise-step-appcenter-ipa-upload/blob/master/step.yml) as a reference. This value must be input in the GUI to run. It is not a default for all apps.

Was I perhaps doing something incorrect?